include/oneapi/dpl/pstl/hetero/algorithm_impl_hetero.h

```diff
@@ -1195,17 +1206,16 @@ __pattern_sort_by_key(_ExecutionPolicy&& __exec, _Iterator1 __keys_first, _Itera
                       _Iterator2 __values_first, _Compare __comp, /*vector=*/::std::true_type,
                       /*parallel=*/::std::true_type)
 {
-    static_assert(::std::is_move_constructible_v<typename ::std::iterator_traits<_Iterator1>::value_type>
-        && ::std::is_move_constructible_v<typename ::std::iterator_traits<_Iterator2>::value_type>,
-        "The keys and values should be move constructible in case of parallel execution.");
+    static_assert(::std::is_move_constructible_v<typename ::std::iterator_traits<_Iterator1>::value_type> &&
```

Comment:
Is this change to formatting intentional or is it the result of applying `clang-format` across the whole file? There are some other instances in this PR where just the formatting has changed like this.